jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -235,6 +235,33 @@ private SchemaVersion getSchemaVersion() throws MojoExecutionException {
         }
     }
 
+    /**
+     * Determine the standard option preset of the generator. Take it from the configuration or set the default.
+     * The default is: PLAIN_JSON
+     *
+     * @return The OptionPreset
+     * @throws MojoExecutionException In case an unknown preset was used.
+     */
+    private OptionPreset getOptionPreset() throws MojoExecutionException {
+        // Unfortunately OptionPreset is not an Enum. So have to do some hardcoding now.
+        // Set the option preset
+        if (options.preset != null && !options.preset.isEmpty()) {
+            switch (options.preset) {
+            case "FULL_DOCUMENTATION":
+                return OptionPreset.FULL_DOCUMENTATION;
+            case "PLAIN_JSON":
+                return OptionPreset.PLAIN_JSON;
+            case "JAVA_OBJECT":
+                return OptionPreset.JAVA_OBJECT;
+            default:
+                throw new MojoExecutionException("Error: Option preset is not one of "
+                        + "['FULL_DOCUMENTATION', 'PLAIN_JSON', 'JAVA_OBJECT']");
+            }
+        }
+
+        return OptionPreset.PLAIN_JSON;
+    }
+
     /**
      * Set the default generator options.
```

Comment:
Let's remove this method and stick to just a default preset.

Reply:
Actually sneaked that into the PR for Unit tests. I'll fix it in this PR as well.

Reply:
or we just merge the other PR and close this one, since all these changes are present in both PRs.

Reply:
Can do that, Can also merge both. 
You choose.